tests/unit/s2n_tls13_pq_handshake_test.c

```diff
@@ -26,31 +26,92 @@
 /* Include C file directly to access static functions */
 #include "tls/s2n_handshake_io.c"
 
-const struct s2n_kem_group *s2n_get_highest_priority_shared_kem_group(const struct s2n_kem_preferences *client_prefs, const struct s2n_kem_preferences *server_prefs)
+const struct s2n_kem_group *s2n_get_predicted_negotiated_kem_group(const struct s2n_kem_preferences *client_prefs, const struct s2n_kem_preferences *server_prefs)
 {
     PTR_ENSURE_REF(client_prefs);
     PTR_ENSURE_REF(server_prefs);
-    for (int i = 0; i < client_prefs->tls13_kem_group_count; i++) {
-        for (int j = 0; j < server_prefs->tls13_kem_group_count; j++) {
-            const struct s2n_kem_group *client_group = client_prefs->tls13_kem_groups[i];
-            const struct s2n_kem_group *server_group = server_prefs->tls13_kem_groups[j];
+
+    /* Client will offer their highest priority PQ KeyShare in their ClientHello. This PQ KeyShare will be most preferred
+     * since it can be negotiated in 1-RTT (even if there are other mutually supported PQ KeyShares that the server would
+     * prefer over this one but would require 2-RTT's). */
```

Comment:
Nit again: Can you use a more conservative line length for comments, since you can easily control comment line length? Something like 100. Your comments are line wrapping for me, which makes them very hard to read. Code can get messy to wrap, but comments are easy.

Reply:
Fixed